arangod/Aql/IResearchViewExecutor.tpp

```diff
@@ -956,6 +1132,15 @@ bool IResearchViewExecutorBase<Impl, ExecutionTraits>::getStoredValuesReaders(
       for (; columnFieldsRegs != columnsFieldsRegs.cend(); ++columnFieldsRegs) {
         TRI_ASSERT(IResearchViewNode::kSortColumnNumber <
                    columnFieldsRegs->first);
+        if constexpr (std::is_same_v<
+                          HeapSortExecutorValue,
+                          typename arangodb::aql::IResearchViewExecutorBase<
+                              Impl,
+                              ExecutionTraits>::Traits::IndexBufferValueType>) {
```

Comment:
Could we just add single constexpr bool instead?
Now this 5 bad formatted lines written few times